### ISSUE TITLE
Clarify intended use of W3C design system

### ISSUE THREAD

**Issue by gosko** [w3c/w3c-website#358]
The [Design System site](https://design-system.w3.org/) has some text that I feel could use some clarification:

> Welcome to the W3C design system
>
> This design system documents the styles, components and templates available to use on your website.

Is it our intent for this design system to be used by others when creating their own sites? My expectation is that it was primarily to be used for W3C's site, though others are welcome to copy and adapt it for their purposes. The [Getting started page](https://design-system.w3.org/getting-started.html) says:

> The assets are automatically compiled and deployed on the W3C CDN under https://www.w3.org/assets/website-2021/. You may link to them directly in any types of projects.

This implies that other sites are welcome to embed these assets directly from W3C's site, which I imagine is not intended even if we do want to encourage others to use these styles.

I think the general issue is that the content of the design system site was written by Studio 24 addressed to W3C staff and in that context the wording makes sense, but since it's public it could use some clarification to avoid people misinterpreting words like "you" and "your" to mean themselves instead of W3C.

Also I think [this text](https://design-system.w3.org/#css):

> CSS is based on [Sass](http://sass-lang.com/) ...

would be better as:

> The CSS for this project is based on [Sass](http://sass-lang.com/) ...

in case that phrase is taken out of context.

@koalie @vivienlacourba I am happy to make these updates to the text directly if you confirm my understanding of the expected use of the design system.

**Comment by vivienlacourba** [comment]
> @koalie @vivienlacourba I am happy to make these updates to the text directly if you confirm my understanding of the expected use of the design system.

@gosko thanks for noticing those and volunteering to update the content. Please work on a PR with edits you see fit and set me as reviewer.

**Comment by koalie** [comment]
> Is it our intent for this design system to be used by others when creating their own sites? My expectation is that it was primarily to be used for W3C's site, though others are welcome to copy and adapt it for their purposes.

Yes, you are correct in your assumption. Therefore, it's indeed best to update the verbiage to match that expectation, in 
>The [Getting started page](https://design-system.w3.org/getting-started.html) says:

>    The assets are automatically compiled and deployed on the W3C CDN under https://www.w3.org/assets/website-2021/. You may link to them directly in any types of projects.

**Comment by japuiglaucia** [comment]
Clarify intended use of W3C design system #358

**Comment by NicolaSaunders** [comment]
> Also I think [this text](https://design-system.w3.org/#css):
>>CSS is based on [Sass](http://sass-lang.com/) ...

> would be better as:
>> The CSS for this project is based on [Sass](http://sass-lang.com/) ...

> in case that phrase is taken out of context.

I just wanted to note that this text has since been updated.
